src/ui/server/controllers/insurance/account/password-reset/success/index.ts

```diff
@@ -0,0 +1,50 @@
+import { PAGES } from '../../../../../content-strings';
+import { ROUTES, TEMPLATES } from '../../../../../constants';
+import insuranceCorePageVariables from '../../../../../helpers/page-variables/core/insurance';
+import { Request, Response } from '../../../../../../types';
+
+export const TEMPLATE = TEMPLATES.INSURANCE.ACCOUNT.PASSWORD_RESET.SUCCESS;
+
+export const PAGE_CONTENT_STRINGS = PAGES.INSURANCE.ACCOUNT.PASSWORD_RESET.SUCCESS;
+
+const {
+  ACCOUNT: {
+    SIGN_IN: { ROOT: SIGN_IN_ROOT },
+    PASSWORD_RESET: { ROOT: PASSWORD_RESET_ROOT },
+  },
+} = ROUTES.INSURANCE;
+
+/**
+ * get
+ * Render the "Password reset - success" page
+ * @param {Express.Request} Express request
+ * @param {Express.Response} Express response
+ * @returns {Express.Response.render} "Password reset - success" page
+ */
+export const get = (req: Request, res: Response) => {
+  const { passwordResetSuccess } = req.session;
+
+  /**
+   * Add the success flag to req.flash
+   * This prevents an issue where the success flag will be undefined if:
+   * - The page is refreshed
+   * - A user navigates away and comes back to the page
+   */
+  req.flash('passwordResetSuccess', String(passwordResetSuccess));
+
+  const success = passwordResetSuccess || req.flash('passwordResetSuccess');
```

Comment:
will the flash always be consumed here? or does it need to be consumed so it is cleared from flash?

Reply:
It won't always be consumed, but solves a scenario where if a user refreshes the page or submits the analytics banner on this page, the success flag will still be defined - so the page will still render.

If however say this URL is visited randomly and there is no success flag, we redirect to the password reset route